plugins/modules/zos_tso_command.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-# Copyright (c) IBM Corporation 2019 - 2023
+# Copyright (c) IBM Corporation 2019 - 2024
```

Comment:
seems like we've been preferring to use commas for copyright: `2019, 2024`

Reply:
done